membrain.cabal

```diff
@@ -50,6 +50,8 @@ common common-options
                        -fhide-source-paths
                        -Wmissing-export-lists
                        -Wpartial-fields
+  if impl(ghc >= 8.8.1)
+    ghc-options:       -Wmissing-deriving-strategies
```

Comment:
@SanchayanMaity while we are on this, could you also please error on missing deriving strategies to make sure they are used? You can see how it's done here in `tomland`:

* https://github.com/kowainik/tomland/blob/ff5506eaa3c1a207f1514a56d12ef377b8cdfaa0/tomland.cabal#L60

Reply:
Pushed. Thank you.